StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  ComponentWebView.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 4/30/24.
+//
+
+@_spi(STP) import StripeCore
+import UIKit
+import WebKit
+
+class ConnectComponentWebView: ConnectWebView {
+    /// The connect instance that will be used for requests.
+    private var connectInstance: StripeConnectInstance
+    
+    /// The component type that should be loaded.
+    private var componentType: ComponentType
+
+    /// The content controller that registers JS -> Swift message handlers
+    private let contentController: WKUserContentController
+
+    /// Represents the current locale that should get sent to the webview
+    private let webLocale: () -> String
+    
+    /// The current notification center instance
+    private let notificationCenter: NotificationCenter
+    
+    init(connectInstance: StripeConnectInstance,
+         componentType: ComponentType,
+         // Should only be overridden for tests
+         notificationCenter: NotificationCenter = NotificationCenter.default,
+         webLocale: @escaping () -> String = { Locale.autoupdatingCurrent.webIdentifier }) {
```

Comment:
I did look into this, but as far as I can tell there isn't a good way of Faking locale's in iOS 15. I figured since for now we are just using the string it is probably best just to mock that. If we use other parts of the locale later I can look into a better solution.

Reply:
You can construct a locale using `Locale(identifier: "fr_FR")`
Example: https://github.com/stripe/stripe-ios/blob/18010dba/Stripe/StripeiOSTests/NSString+StripeTest.swift#L73